crates/bevy_app/src/system_registry.rs

```diff
@@ -0,0 +1,68 @@
+use crate::App;
+use bevy_ecs::prelude::*;
+use bevy_ecs::system::SystemRegistryError;
+
+impl App {
+    /// Registers the supplied system in the [`SystemRegistry`](bevy_ecs::system::SystemRegistry) resource.
+    ///
+    /// Calls the method of the same name on [`SystemRegistry`](bevy_ecs::system::SystemRegistry).
+    #[inline]
+    pub fn register_system<Params, S: IntoSystem<(), (), Params> + 'static>(
+        &mut self,
+        system: S,
+    ) -> &mut Self {
+        self.world.register_system(system);
+        self
+    }
+
+    /// Register system a system with any number of [`SystemLabel`]s.
+    ///
+    /// Calls the method of the same name on [`SystemRegistry`].
```

Comment:
```suggestion
    /// Calls the method of the same name on [`SystemRegistry`].
    /// 
    /// [`SystemRegistry`]: bevy_ecs::system::SystemRegistry
```

There was an undefined link. It should also make it pass CI.